### PR TITLE
Uxd 756 title prop should be optional

### PR DESCRIPTION
**Pull request by allison-c**
### Purpose 🚀
_clear, one sentence description of primary purpose of this PR_

### Notes ✏️

I have to hardcode the props parameter in a template to make the default value to `null`. SVGR doesn't have a configuration for it.

Also I noticed there's no js files in src folder so I deleted the ts files too

Will do the same thing in wasabicons 😺 

### Updates 📦
- [ ] MAJOR (breaking) change to _these packages_
- [ ] MINOR (backward compatible) change to _these packages_
- [x] PATCH (bug fix) change to `@paprika/icon`

### Storybook 📕
http://storybooks.highbond-s3.com/paprika/your-branch-name

### Screenshots 📸
_optional but highly recommended_

### References 🔗
_relevant Jira ticket / GitHub issues_

### CODE REVIEW

File: packages/Icon/svgr.config.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  svgProps: {
+    // We need to render string template
+    // eslint-disable-next-line no-template-curly-in-string
+    css: "{`color: ${props.color};width: ${props.size};height: ${props.size};vertical-align: text-top;`}",
+    "data-pka-anchor": "icon",
+    focusable: false,
+  },
+  titleProp: true,
+  template,
+};
```
Comment (allison-c) [inline packages/Icon/svgr.config.js:21]: `.svgrrc` cannot handle template correctly, so uses `svgr.config.js` instead

File: packages/Icon/helper/template.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+module.exports = ({ template }, opts, { imports, interfaces, componentName, jsx, exports }) => {
```
Comment (nahumzs) [inline packages/Icon/helper/template.js:1]: maybe would worth putting a link or reference where to read more about it? so the next developer can get an idea of how this works?

**Review by nahumzs — APPROVED** [review]

File: packages/Icon/helper/template.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  const plugins = ["jsx"];
+  const typeScriptTpl = template.smart({ plugins });
+  return typeScriptTpl.ast`
+  ${imports}
+  ${interfaces}
+  function ${componentName}({title=null, ...props}) {
```
Comment (nahumzs) [inline packages/Icon/helper/template.js:7]: I guess this is trigger by typescript and modified the `ast` while executing the code? or how does this works?

Comment (allison-c) [reply packages/Icon/helper/template.js:7]: This triggered by svgr plugin https://github.com/gregberge/svgr/blob/master/packages/babel-plugin-transform-svg-component/src/index.js
You can find the default template on that page as well

**Pull request merged**